tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Azure.Sdk.Tools.CheckEnforcer.csproj

```diff
@@ -7,6 +7,7 @@
     <PackageReference Include="Azure.Identity" Version="1.0.0-preview.4" />
     <PackageReference Include="Azure.Security.KeyVault.Keys" Version="4.0.0-preview.4" />
     <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.0.0-preview.4" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0-preview.3" />
```

Comment:
did you not want to use preview.4 for storage like the others?

Reply:
Updated - thanks - I had meant to do that.

Reply:
Ended up reverting that change due to a compat issue. The functions team is aware.